gcc/rust/hir/rust-ast-lower-type.cc

```diff
@@ -445,17 +446,19 @@ ASTLowerGenericParam::translate (AST::GenericParam &param)
 void
 ASTLowerGenericParam::visit (AST::LifetimeParam &param)
 {
+  AST::Attribute outer_attr = AST::Attribute::create_empty ();
```

Comment:
I think we can remove this since we are not using it? or maybe I don't see the use

Reply:
Fixed,

I thought the compiler would emit warnings for this, no?

Reply:
I think it does not emit a warning because the constructor has a side effect, it increments the node id.